main.go

```diff
@@ -16,9 +14,10 @@ import (
 	"strings"
 	"text/template"
 
+	"path"
```

Comment:
This doesn't need to be separate from the imports above it, does it?

Reply:
Nope, and in fact I'm surprised this didn't get gofmt'd into a nicer shape. Will fix.

Reply:
Fixed.